gadgets/advise_seccomp/README.mdx

```diff
@@ -0,0 +1,485 @@
+---
+title: advise_seccomp
+sidebar_position: 0
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The seccomp profile advisor gadget records syscalls that are issued in a
+specified container, and then uses this information to generate the corresponding
+seccomp profile.
+
+## Getting started
+
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
+        ```bash
+        $ kubectl gadget run ghcr.io/inspektor-gadget/gadget/advise_seccomp:%IG_TAG% [flags]
+        ```
+    </TabItem>
+
+    <TabItem value="ig" label="ig">
+        ```bash
+        $ sudo ig run ghcr.io/inspektor-gadget/gadget/advise_seccomp:%IG_TAG% [flags]
+        ```
+    </TabItem>
+</Tabs>
+
+## Flags
+
+No Flags.
+
+## Guide
```

Comment:
Additional sections: `## Limitations`:
* Does it start recording at the right point in the runc startup?
* Does it work with crun?
* General warning that it is difficult to exercise all code paths in your workload
* Run workload without seccomp filter during the learning phase to be able to capture all syscalls (see comment below).
* Merging profiles with ReplicaSet

Additional sections: `## Related project`:
* SPO's Profile Recording: https://github.com/kubernetes-sigs/security-profiles-operator/blob/main/installation-usage.md#record-seccomp-profile
* https://github.com/containers/oci-seccomp-bpf-hook (podman only)